### PR TITLE
Added IF EXISTS for DROP DATABASE command, #4659

### PR DESCRIPTION
**Pull request by ch33hau**
Hi,

Can I create a pull request for this issue? https://github.com/influxdb/influxdb/issues/4659

Thanks

### CODE REVIEW

File: tsdb/query_executor.go [hunk 1]
```diff
@@ -369,6 +369,8 @@ func (q *QueryExecutor) executeDropDatabaseStatement(stmt *influxql.DropDatabase
 	dbi, err := q.MetaStore.Database(stmt.Name)
 	if err != nil {
 		return &influxql.Result{Err: err}
+	} else if dbi == nil && stmt.IfExists {
```
Comment (otoolep) [inline tsdb/query_executor.go]: Minor, but I think better style would be:

```
} else if dbi == nil {
    if stmt.IfExists {
        return &influxql.Result{}
    }
    return &influxql.Result{Err: ErrDatabaseNotFound(stmt.Name)}
}
```

**Comment by otoolep** [comment]
Thanks @ch33hau -- looks great. I have some small feedback before we merge.

@dgnorton @corylanou  -- can I get a second review?

**Comment by corylanou** [comment]
Looks good.  +1

File: tsdb/query_executor.go (hunk 1 above)
Comment (ch33hau) [inline tsdb/query_executor.go]: Hi @otoolep , Thanks for your suggestion, I have updated it.

**Comment by ch33hau** [comment]
Thanks @otoolep  @corylanou  =)

**Pull request merged**